spec/features/further_education_payments/provider/provider_verifying_claims_spec.rb

```diff
@@ -2,6 +2,11 @@
 
 RSpec.feature "Provider verifying claims" do
   before do
+    FeatureFlag.create!(
+      name: "fe_provider_identity_verification",
+      enabled: true
+    )
```

Comment:
fyi I added this https://github.com/DFE-Digital/claim-additional-payments-for-teaching/blob/master/spec/support/feature_flags.rb so you can tag your test with
```
scenario "with flag", feature_flag: :fe_provider_identity_verification do
```